package.json

```diff
@@ -45,8 +45,10 @@
     "@vibrant/color": "^3.2.1-alpha.1",
     "@vibrant/generator-default": "^3.2.1-alpha.1",
     "@vibrant/quantizer-mmcq": "^3.2.1-alpha.1",
+    "buffer": "^6.0.3",
     "id3-parser": "^2.0.0",
     "idb-keyval": "^5.0.6",
+    "music-metadata": "^7.11.3",
```

Comment:
You could possibly use import [`music-metadata-browser`](https://github.com/Borewit/music-metadata-browser) and pass [`Response.blob()`](https://developer.mozilla.org/en-US/docs/Web/API/Response/blob) to [`parseblob()`](https://github.com/Borewit/music-metadata-browser#parseblob-function)